plugins/coriolis.py

```diff
@@ -13,13 +22,101 @@
 
 config.delete('shipyard', suppress=True)
 
+logger = get_main_logger()
+
+DEFAULT_NORMAL_URL = 'https://coriolis.io/import?data='
+DEFAULT_BETA_URL = 'https://beta.coriolis.io/import?data='
+DEFAULT_OVERRIDE_MODE = 'auto'
```

Comment:
This starts out as 'auto' in the dropdown, but in the dropdown there's "normal", "beta" and **"default"**.  So one of 'auto' or 'default' needs to change.

Reply:
whoops, it should be auto

Reply:
Changed

Reply:
s/auto/Auto/ - else first time you look it shows this "auto" as the current value, when it's "Auto" in the dropdown.